src/webgl/p5.Framebuffer.js

```diff
@@ -1489,16 +1489,17 @@ class Framebuffer {
    *   createCanvas(400, 400, WEBGL);
    *
    *   // Create an off-screen WebGL graphics buffer
-   *   let myBuffer = createGraphics(200, 200, WEBGL);
-   *
+   *   let myBuffer = createFramebuffer({ width: 200, height: 200 });
    *   // Draw a red box on the off-screen buffer
-   *   myBuffer.background(0);  // Set the background to black
-   *   myBuffer.noStroke();
-   *   myBuffer.fill(255, 0, 0);  // Set the fill color to red
-   *   myBuffer.push();
-   *   myBuffer.translate(0, 0, 0);
-   *   myBuffer.box(100);  // Draw a red box at the center
-   *   myBuffer.pop();
+   *   myBuffer.draw(() => {
+   *     background(0); // Set the background to black
+   *     noStroke();
+   *     fill(255, 0, 0); // Set the fill color to red
+   *     push();
+   *     translate(0, 0, 0);
```

Comment:
We maybe don't need the push/translate/pop since the translation is 0 here, but otherwise this is looking good!

Reply:
thank you